### PR TITLE
add home_verse

### PR DESCRIPTION
**Pull request by phi7**
Create eip155-19011.json
Create home_verse.json

Thank you!

### CODE REVIEW

File: _data/chains/eip155-19011.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "decimals": 18
+    },
+    "infoURL": "https://www.homeverse.games/",
+    "shortName": "HMV",
+    "chainId": 19011,
+    "networkId": 19011,
+    "explorers": [
+      {
+        "name": "HOME Verse Explorer",
+        "url": "https://explorer.oasys.homeverse.games/",
```
Comment (ligi) [inline _data/chains/eip155-19011.json:19]:
```suggestion
        "url": "https://explorer.oasys.homeverse.games",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by phi7** [comment]
I run the prerinter!

File: _data/chains/eip155-19011.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://www.homeverse.games/",
+  "shortName": "HMV",
+  "chainId": 19011,
+  "networkId": 19011,
+  "explorers": [
+    {
+      "name": "HOME Verse Explorer",
+      "url": "https://explorer.oasys.homeverse.games/",
```
Comment (ligi) [inline _data/chains/eip155-19011.json:19]:
```suggestion
      "url": "https://explorer.oasys.homeverse.games",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**